packages/babel-traverse/src/index.js

```diff
@@ -114,3 +114,6 @@ traverse.hasType = function (
 };
 
 traverse.cache = cache;
+
+export default traverse;
+export { traverse };
```

Comment:
`export { traverse as default }` is required to create a live binding:
```suggestion
export { traverse as default };
```

Using `export default <expression>` creates a snapshot of the `expression`.